org-edit-latex.el

```diff
@@ -297,6 +297,20 @@ latex-environment."
           (apply oldfun args)))
     (apply oldfun args)))
 
+;;;###autoload
+(defun org-edit-latex-smart-hint ()
+  (if (and (equal major-mode 'org-mode)
+           (or (eq 'latex-environment (car (org-element-context)))
+               (org-inside-LaTeX-fragment-p)))
```

Comment:
BTW, can `org-edit-latex` support this inline style LaTeX fragment? Hope can add support for this.